ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/StreamedQueryResult.java

```diff
@@ -90,4 +93,9 @@
    */
   boolean isFailed();
 
+  boolean hasContinuationToken();
+
+  Optional<AtomicReference<String>> getContinuationToken();
```

Comment:
We could also just expose a single getter method like this, and assume the optional will be empty if there's no token.  At the moment, it's always there, from what I recall.

Reply:
Also, this method can probably just return an Optional<String>, hiding the fact that it's held by an AtomicReference.

Reply:
i just went ahead and tested removing the optional all together now and it worked. This is also only now exposed in the impl class anyway